src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -807,7 +807,7 @@
 type: command
 short-summary: Add a timer trigger to a task.
 examples:
-  - name: Add a timer trigger to a task.
+  - name: Add a timer trigger at 09:30 (UTC) on every day-of-week from Monday through Friday.
```

Comment:
Minor rephrase - 
Add a timer trigger which schedules the task at 09:30 (UTC)  every day
of the week from Monday through Friday.